### PR TITLE
Move Tested Work from Develop to Staging

### PR DESCRIPTION
**Pull request by Andretalik**
## Description (what problem you're fixing)

  - The work that has been done needs to be moved from develop to staging.

## Fix (what you did to fix it)

  - N/A

## How to test (describe how to test your PR)

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.

### CODE REVIEW

**Comment by LarryWachira** [comment]
@Andretalik Would you be so kind as to resolve the conflicts?

**Comment by georgreen** [comment]
@Andretalik @LarryWachira staging should be an exact copy of develop, so you can overwrite staging without resolving conflicts.

**Comment by LarryWachira** [comment]
Got it @georgreen ! But then what really are we reviewing?

**Comment by WinstonKamau** [comment]
@LarryWachira  Currently backend points to commit d15f52c4d499957e23769f6272978941db3ea98f

File: src/api/endpoints/redemption_requests.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
+"""RedemptionRequest Module."""
+
+from flask import request, g, current_app
+from flask_restplus import Resource
```
Comment (LarryWachira) [inline src/api/endpoints/redemption_requests.py]: `flask_restful`

**Comment by LarryWachira** [comment]
Thanks @WinstonKamau , that helped a lot.

**Comment by kimani-ndegwa** [comment]
Can we resolve the conflicts here?

**Comment by georgreen** [comment]
@LarryWachira I will reset staging to be one commit behind develop then, merge develop to staging.

**Comment by LarryWachira** [comment]
Much better. Then it's good to go now @georgreen . Merge mercilessly. 🤣

**Pull request merged**